databricks/koalas/tests/test_dataframe_spark_io.py

```diff
@@ -375,3 +377,58 @@ def test_read_excel(self):
                     )
             else:
                 self.assertRaises(ValueError, lambda: ks.read_excel(tmp))
+
+    def test_read_orc(self):
+        with self.temp_dir() as tmp:
+            path = "{}/file1.orc".format(tmp)
+            data = self.test_pdf
+            self.spark.createDataFrame(data, "i32 int, i64 long, f double, bhello string").coalesce(
+                1
+            ).write.orc(path, mode="overwrite")
+
+            # `spark.write.orc` create a directory contains distributed orc files.
+            # But pandas only can read from file, not directory. Therefore, we need orc file path.
+            orc_file_path = glob.glob(os.path.join(path, "*.orc"))[0]
+
+            if LooseVersion(pd.__version__) >= LooseVersion("1.0.0"):
+                expected = pd.read_orc(orc_file_path)
+            else:
+                expected = data.reset_index()[data.columns]
+            actual = ks.read_orc(path)
+            self.assertPandasEqual(expected, actual.to_pandas())
+
+            # columns
+            columns = ["i32", "i64"]
+            if LooseVersion(pd.__version__) >= LooseVersion("1.0.0"):
+                expected = pd.read_orc(orc_file_path, columns=columns)
+            else:
+                expected = data.reset_index()[columns]
+            actual = ks.read_orc(path, columns=columns)
+            self.assertPandasEqual(expected, actual.to_pandas())
+
+            # index_col
+            kdf = ks.from_pandas(data)
+            expected = kdf.set_index("i32")
```

Comment:
Why do we use Koalas objects for expected?

Reply:
Fixed it to pandas one. Thanks!